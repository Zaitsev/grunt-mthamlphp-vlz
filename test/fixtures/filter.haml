:less
    a{
        color:#0000FF
    }
#div.a
:coffee
    ()->
        return
:markdown
    __italic__