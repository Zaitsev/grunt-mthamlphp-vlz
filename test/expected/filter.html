<style type="text/css">
a {
  color: #0000FF;
}

</style>
<div id="div" class="a"></div>
<script type="text/javascript">
// Generated by CoffeeScript PHP 1.3.1
(function() {

  (function() {});

}).call(this);

</script>
<p><strong>italic</strong></p>